stratosphere/pm/source/pm_info.cpp

```diff
@@ -15,16 +15,36 @@
  */
 
 #include <switch.h>
+#include <stratosphere/ldr.hpp>
+#include <stratosphere/ldr/ldr_pm_api.hpp>
+
 #include "pm_registration.hpp"
 #include "pm_info.hpp"
 
 Result InformationService::GetTitleId(Out<u64> tid, u64 pid) {
     std::scoped_lock<ProcessList &> lk(Registration::GetProcessList());
 
     std::shared_ptr<Registration::Process> proc = Registration::GetProcess(pid);
-    if (proc != NULL) {
-        tid.SetValue(proc->tid_sid.title_id);
-        return ResultSuccess;
+    if (proc == NULL) {
```

Comment:
```suggestion
    if (proc == nullptr)
```

Reply:
Yep, will fix when PM is rewritten following this PR being merged.